config/default.settings.local.php

```diff
@@ -13,18 +13,18 @@
  */
 
 // Database settings.
-$databases = array (
+$databases = array(
   'default' =>
-    array (
+    array(
```

Comment:
I think it's worth adding it since it improves readability by a ton. A lot of our more recent code has been using short-array syntax as well so it's kinda all over the place right now.